platforms/src/GtcStaking/__tests__/communityStaking.test.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
Do we really need this eslint-disable?

Reply:
I suppose we don't, but I wasn't able to commit my changes until I added it. Is there another flag we can use to bypass eslint for now?

Reply:
We should not bypass eslint, but rather fix the linter errors.